docs/admin/extensible-admission-controllers.md

```diff
@@ -31,34 +30,34 @@ This page describes how to use Initializers and External Admission Webhooks.
 
 ### What are initializers?
 
-Two meanings:
+*Initializer* has two meanings:
 
 * A list of pending pre-initialization tasks, stored in every object's metadata
   (e.g., "AddMyCorporatePolicySidecar").
 
-* The controllers which actually perform those tasks. The name of the task
+* A controller, which actually perform those tasks. The name of the task
```

Comment:
Users are supposed to write these controllers. Is this message clear? Or shall we s/A controller/a user costomized controller/?